app/lib/search/index_simple.dart

```diff
@@ -73,11 +81,8 @@ class SimplePackageIndex implements PackageIndex {
       addAll(_nameIndex.search(query.packagePrefix), 0.8);
     }
 
-    final Map<String, double> popularityScores = new Map.fromIterable(
-      total.keys,
-      value: (String url) => _documents[url].popularity * 100,
-    );
-    addAll(popularityScores, 0.05);
+    addAll(getHealthScore(total.keys), 0.05);
+    addAll(getPopularityScore(total.keys), 0.05);
```

Comment:
How much would you attribute to it, and which part of the score would you decrease?